File: questions/exam/autoR/regionCheapestAvocados/tests/ans.R
library(tidyverse)

avocado <- read_csv("/grade/serverFilesCourse/avocado_prices.csv")

cheapest <- avocado |>
    filter(yr == 2018) |> 
    group_by(region) |>
    summarize(avg_price = mean(average_price)) |>
    arrange(avg_price) |> 
    slice(1)

File: questions/exam/autoPy/regionCheapestAvocados/tests/ans.py
import pandas as pd

avocado = pd.read_csv("/grade/serverFilesCourse/dataR/avocado_prices.csv")

cheapest = (
    (
        avocado[avocado["yr"] == 2018]
        .groupby("region")
        .mean(numeric_only=True)
        .reset_index()
        .sort_values(by="average_price")
    )
    .head(1)
    [["region", "average_price"]]
)